### ISSUE TITLE
Cannot read property 'paths' of undefined

### ISSUE THREAD

**Issue by ViniGodoy** [cdimascio/express-openapi-validator#416]
**Describe the bug**

I am getting the following error, in any route:
Cannot read property 'paths' of undefined
   at defaultResolver (/Users/vinigodoy/Documents/git/clustersrv/api/node_modules/express-openapi-validator/dist/resolvers.js:10:27)
    at OpenApiValidator.installOperationHandlers (/Users/vinigodoy/Documents/git/clustersrv/api/node_modules/express-openapi-validator/dist/openapi.validator.js:249:52)
    at /Users/vinigodoy/Documents/git/clustersrv/api/node_modules/express-openapi-validator/dist/openapi.validator.js:167:59

The bug is on instalOperationHandlers method, in openapi.validator.js line 249. It is:

`router[method.toLowerCase()](path, resolver(basePath, route));`

The resolver variable is filled with the defaultResolver function. It demands a third parameter, the apiDocs.
Notice that the third parameter (apiDoc) is missing. 

The correct call must be:

`router[method.toLowerCase()](path, resolver(basePath, route, this.options.apiSpec));`

I've tried it and this solved the problem. I've tested this solution in OpenAPI 4.3.1

**Comment by cdimascio** [comment]
thanks @ViniGodoy fixed, in `v4.3.2`

**Comment by ViniGodoy** [comment]
Thanks, that was dazzling fast.

I have no idea why this code worked before. I've been using this module for the entire month. I thought I the bug could be a due to the fact I recently updated from 4.2.0 to 4.3.2. But then I tried to revert to 4.2.0 and the bug was still there! And even running with this version, that was installed and running in my production server before, the bug appeared!

I had made no changed in operation handlers configuration. So, I don't have the faintest idea why the problem started appearing. Maybe a deeper investigation is necessary to solve this mistery.
